Resources/Scripts/playgrounds_app_link.js

```diff
@@ -29,6 +31,11 @@ export class SPIPlaygroundsAppLinkFallback {
         setTimeout(() => {
           explainerElement.classList.add('hidden')
         }, 5000)
+
+        // Trigger an analytics event and pass the dependency as a custom property
+        const targetUrl = new URL(linkElement.href)
+        const dependencies = targetUrl.searchParams.get('dependencies')
+        measurePlausibleEvent('SPI Playgrounds Launch', { props: { dependency: dependencies } })
```

Comment:
Deliberately using the singular `dependency` here so we can measure this in a different way if/when we support multiple dependencies.

Reply:
So is this reporting on any url with `dependencies=` as a parameter? Should we also look for `spi-playgrounds://open` in the url? Or is this configured server side somewhere?

Reply:
This code only executes on those links, that check is further up ^^

Reply:
Oh, I see, it was already in and not part of the diff :+1: